sdk/openai/openai/test/public/openai.spec.ts

```diff
@@ -2,36 +2,43 @@
 // Licensed under the MIT license.
 
 import { Recorder } from "@azure-tools/test-recorder";
-import { assert } from "@azure/test-utils";
+import { assert, isNode, matrix } from "@azure/test-utils";
 import { Context } from "mocha";
 import { OpenAIClient } from "../../src/OpenAIClient.js";
-import { createClient, startRecorder } from "./utils/recordedClient.js";
+import { AuthMethod, createClient, startRecorder } from "./utils/recordedClient.js";
 
-describe("client", () => {
-  let recorder: Recorder;
-  let client: OpenAIClient;
+matrix([["AzureAPIKey", "AAD", "OpenAIKey"]] as const, async function (authMethod: AuthMethod) {
+  describe(`[${authMethod}] Client`, () => {
+    let recorder: Recorder;
+    let client: OpenAIClient;
 
-  beforeEach(async function (this: Context) {
-    recorder = new Recorder(this.currentTest);
-    recorder = await startRecorder(this.currentTest);
-    client = createClient("AzureAPIKey", { recorder });
-  });
+    beforeEach(async function (this: Context) {
+      if (!isNode && authMethod == "AAD") {
+        this.skip();
+      }
+      recorder = new Recorder(this.currentTest);
+      recorder = await startRecorder(this.currentTest);
+      client = createClient(authMethod, { recorder });
+    });
 
-  afterEach(async function () {
-    await recorder.stop();
-  });
+    afterEach(async function () {
+      if (!isNode && authMethod == "AAD") {
+        this.skip();
+      }
+      await recorder.stop();
+    });
 
-  it("completions test", async function () {
-    const prompt = ["This is a test"];
-    const modelName = "text-davinci-003";
-    const completions = await client.getCompletions(modelName, prompt);
-    assert.isNotNull(completions.choices);
-    assert.equal(completions.choices?.length, 1);
-  });
+    it("completions test", async function () {
+      const prompt = ["This is a test"];
```

Comment:
format

Reply:
Are you referring to the extra indentation? it is because of the additional level the `matrix` call introduces.